# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/glut/no-m486.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./linux/Makefile.orig	2004-10-04 00:13:52.812539096 +0000
+++ ./linux/Makefile	2004-10-04 00:15:42.650841136 +0000
@@ -104,11 +104,11 @@
 .SUFFIXES: .cxx
 
               CXX = g++
-    CXXDEBUGFLAGS = -O2 -m486
+    CXXDEBUGFLAGS = -O2
  CXXEXTRA_DEFINES =
 CXXEXTRA_INCLUDES =
   CXXIDL_INCLUDES = -I$(TOP)/include
-   CXXSTD_DEFINES = -Dlinux -D__i386__ -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
+   CXXSTD_DEFINES = -Dlinux -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
        CXXOPTIONS =
       CXXINCLUDES = $(INCLUDES) $(TOP_INCLUDES) $(CXXEXTRA_INCLUDES) $(CXXIDL_INCLUDES)
        CXXDEFINES = $(CXXINCLUDES) $(CXXSTD_DEFINES) $(THREADS_CXXDEFINES) $(CXXEXTRA_DEFINES)
@@ -148,8 +148,8 @@
             LATEX = latex
 
      STD_INCLUDES =
-  STD_CPP_DEFINES = -traditional -Dlinux -D__i386__ -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
-      STD_DEFINES = -Dlinux -D__i386__ -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
+  STD_CPP_DEFINES = -traditional -Dlinux -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
+      STD_DEFINES = -Dlinux -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
  EXTRA_LOAD_FLAGS = -Wl,-rpath,/usr/X11R6/lib
   EXTRA_LDOPTIONS =
   EXTRA_LIBRARIES =
@@ -180,7 +180,7 @@
 
      TOP_INCLUDES = -I$(INCROOT)
 
-      CDEBUGFLAGS = -O2 -m486
+      CDEBUGFLAGS = -O2
         CCOPTIONS = -ansi
 
       ALLINCLUDES = $(INCLUDES) $(EXTRA_INCLUDES) $(TOP_INCLUDES) $(STD_INCLUDES)
@@ -473,8 +473,8 @@
   CCOPTIONS = -ansi -fPIC
 
 
-STD_DEFINES = -Dlinux -D__i386__ -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
-CDEBUGFLAGS = -O2 -m486
+STD_DEFINES = -Dlinux -D_POSIX_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -DX_LOCALE
+CDEBUGFLAGS = -O2
 
 # Copyright (c) Mark J. Kilgard, 1994.
 # Glut.cf - GLUT distribution Imakefile configuration info
